test/processor/ConfigurationProcessor.test.ts

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+import fs from 'fs-extra';
+import { ConfigurationProcessor } from '../../src/processor/ConfigurationProcessor';
+import { ExportableConfiguration } from '../../src/exportable';
+import { loggerSpy } from '../helpers/loggerSpy';
+
+describe('ConfigurationProcessor', () => {
+  it('should create a Configuration from an ImplementationGuide with url and fhirVersion properties', () => {
+    const input = JSON.parse(
+      fs.readFileSync(path.join(__dirname, 'fixtures', 'simple-ig.json'), 'utf-8')
+    );
+    const result = ConfigurationProcessor.process(input);
+    expect(result).toBeInstanceOf(ExportableConfiguration);
+    expect(result.config.canonical).toBe('http://example.org/tests');
+    expect(result.config.fhirVersion).toEqual(['4.0.1']);
+  });
+
+  it('should not create a Configuration without a url', () => {
+    const input = JSON.parse(
+      fs.readFileSync(path.join(__dirname, 'fixtures', 'missing-url-ig.json'), 'utf-8')
+    );
+    const result = ConfigurationProcessor.process(input);
+    expect(result).toBeUndefined();
+    expect(loggerSpy.getLastMessage('warn')).toMatch(
+      /ImplementationGuide missing properties.*url/s
+    );
+  });
+
+  it('should not create a Configuration with a fhirVersion', () => {
```

Comment:
I think this should say: "should not create a Configuration **without** a fhirVersion".